src/fsharp/NicePrint.fs

```diff
@@ -1301,6 +1301,29 @@ module private PrintTastMemberOrVals =
     let prettyLayoutOfMemberNoInstShort denv v = 
         prettyLayoutOfMemberShortOption denv emptyTyparInst v true |> snd
 
+    let prettyLayoutOfLiteralValue literalValue =
+        let literalValue =
+            match literalValue with
+            | Const.Bool value -> if value then WordL.keywordTrue else WordL.keywordFalse
+            | Const.SByte _
+            | Const.Byte _
+            | Const.Int16 _
+            | Const.UInt16 _
+            | Const.Int32 _
+            | Const.UInt32 _
+            | Const.Int64 _
+            | Const.UInt64 _
+            | Const.IntPtr _
+            | Const.UIntPtr _
+            | Const.Single _
+            | Const.Double _
+            | Const.Decimal _ -> literalValue.ToString() |> tagNumericLiteral |> wordL
+            | Const.Char _
+            | Const.String _ -> literalValue.ToString() |> tagStringLiteral |> wordL
+            | Const.Unit  -> literalValue.ToString() |> tagText |> wordL
+            | Const.Zero -> literalValue.ToString() |> tagText |> wordL
+        LeftL.leftParen ++ literalValue ++ RightL.rightParen
```

Comment:
I think a ` ... = <value>` approach would be a little better here

Reply:
changed to = <value>